draft-ietf-quic-recovery.md

```diff
@@ -690,12 +690,14 @@ losses or increases in the ECN-CE counter.
 
 ## Ignoring Loss of Undecryptable Packets	
 
-During the handshake, some packet protection keys might not be	
-available when a packet arrives. In particular, Handshake and 0-RTT packets	
-cannot be processed until the Initial packets arrive, and 1-RTT packets	
-cannot be processed until the handshake completes.  Endpoints MAY	
-ignore the loss of Handshake, 0-RTT, and 1-RTT packets that might arrive before	
-the peer has packet protection keys to process those packets.	
+During the handshake, some packet protection keys might not be available when
+a packet arrives and the receiver can choose to drop the packet. In particular,
+Handshake and 0-RTT packets cannot be processed until the Initial packets
+arrive and 1-RTT packets cannot be processed until the handshake completes.
+Endpoints MAY ignore the loss of Handshake, 0-RTT, and 1-RTT packets that might
+have arrived before the peer has packet protection keys to process those
+packets. Endpoints MUST NOT ignore the loss of packets that were sent after
+the earliest acknowledged packet in a given packet number space.
 
```

Comment:
How would that (not) impact the congestion controller?

Reply:
I'm unsure what you're asking?

Reply:
If you send a packet you consume bytes in flight. If you do not get an ack for that packet, eventually it is lost. I'm not sure what it means to ignore in that context unless you complete ignore an earlier packet space. But perhaps the text refers to ignoring received packets, not sent packets. In that case I would say drop, not ignore.

Reply:
#3415 makes that extra clear I think.